src/lsp/NotificationHandlers.re

```diff
@@ -17,7 +17,7 @@ let checkPackageTimers = state => {
   Hashtbl.iter((_, package) => {
     if (package.rebuildTimer != 0. && package.rebuildTimer < Unix.gettimeofday()) {
       package.rebuildTimer = 0.;
-      State.runBuildCommand(state, package.basePath, package.buildCommand);
+      ignore @@ State.runBuildCommand(state, package.basePath, package.buildCommand);
```

Comment:
umm how bout you report this error? :P with a [showMessage notification](https://microsoft.github.io/language-server-protocol/specification#window_showMessage)